src/components/new-request/search.vue

```diff
@@ -739,17 +754,17 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     }
   }
 
-  /** Returns whether the selected XPRO is restorable. */
```

Comment:
Small technical point: this doesn't "return" anything (like a function would). Instead, it "is" a value.

(FYI, getters aka computed are evaluated the first time they're needed, then they're cached until something in them changes. So they can be faster than a function!)